OpenEdXMobile/res/layout/fragment_learn.xml

```diff
@@ -33,25 +33,23 @@
                 <LinearLayout
                     android:id="@+id/ll_learn_selection"
                     android:layout_width="wrap_content"
```

Comment:
Should be set to `match_parent` to support RTL.

Reply:
same here.